src/components/Navbar/Navbar.jsx

```diff
@@ -46,31 +48,37 @@ const Navbar = () => {
                         />
                     </div>
                     <ul className="navbar-nav navbar-fixed">
-                       
+
                         <li className="nav-item">
-                                <span className="nav-icon-link no-link-style"
-                                onClick={() => token
-                                    ? navigate('/Cart')
-                                    : navigate('Login')}
+                            <div className="nav-icon-link no-link-style">
+                                <span className="nav-icon badge-container"
+                                    onClick={() => token
+                                        ? navigate('/Cart')
+                                        : navigate('Login')}
                                 >
                                     <i className="fas fa-shopping-bag"></i>
+                                    {cart.length>0 && <span class="btn-badge d-flex justify-center items-center">{cart.length}</span>}
                                 </span>
+                            </div>
                         </li>
                         <li className="nav-item">
-                                <span className="nav-icon-link no-link-style"
-                                onClick={() => token
-                                    ? navigate('/Wishlist')
-                                    : navigate('Login')}
-                                >   
+                            <div className="nav-icon-link no-link-style">
+                                <span className="nav-icon badge-container"
+                                    onClick={() => token
+                                        ? navigate('/Wishlist')
+                                        : navigate('Login')}
+                                >
                                     <i className="fas fa-heart"></i>
+                                    <span class="btn-badge d-flex justify-center items-center">8</span>
                                 </span>
+                            </div>
                         </li>
                         <li className="nav-item">
                             <Link to='/Login' className='nav-item-link'>LOGIN</Link>
                         </li>
-                        <li className="nav-item">
+                        {/* <li className="nav-item">
```

Comment:
Just in case you forgot - you can remove commented out code.

Reply:
I forgot to remove that. Will do it.